changelog.md

```diff
@@ -66,7 +66,8 @@
 
 - Added `nim --eval:cmd` to evaluate a command directly, see `nim --help`.
 
-
+- `cast` now works in VM in more contexts, behaving like in RT; eg: `int <=> ptr T | ref T`,
```

Comment:
Jargon: Not many know what "RT" is supposed to mean.

Reply:
What is `int <=> ptr T | ref T` ? (Yes, I do know what it means.) 

If you cannot find the time to type, why am I supposed to find the time to review your PRs?

Reply:
> Not many know what "RT" is supposed to mean

rephrased it. Although I think CT and RT should become standard terminology, it's often used.

> `int <=> ptr T | ref T`

rephrased. Although `int <=> ptr T | ref T` was IMO precise and clear in the context of cast.